doc/releases/changelog-0.37.0.md

```diff
@@ -103,75 +103,79 @@
-  mid-circuit measurements.
+* `qml.QNode` and the `@qml.qnode` decorator now accept two new keyword arguments: `postselect_mode` 
+  and `mcm_method`. These keyword arguments can be used to configure how the device should behave when 
+  running circuits with mid-circuit measurements.
   [(#5679)](https://github.com/PennyLaneAI/pennylane/pull/5679)
   [(#5833)](https://github.com/PennyLaneAI/pennylane/pull/5833)
   [(#5850)](https://github.com/PennyLaneAI/pennylane/pull/5850)
 
-  * `postselect_mode="hw-like"` will indicate to devices to discard invalid shots when postselecting
+  * `postselect_mode="hw-like"` indicates to devices to discard invalid shots when postselecting
     mid-circuit measurements. Use `postselect_mode="fill-shots"` to unconditionally sample the postselected
     value, thus making all samples valid. This is equivalent to sampling until the number of valid samples
     matches the total number of shots.
   * `mcm_method` will indicate which strategy to use for running circuits with mid-circuit measurements.
     Use `mcm_method="deferred"` to use the deferred measurements principle, or `mcm_method="one-shot"`
-    to execute once for each shot. If using `qml.jit` with the Catalyst compiler, `mcm_method="single-branch-statistics"`
+    to execute once for each shot. If `qml.qjit` is being used (the Catalyst compiler), `mcm_method="single-branch-statistics"`
     is also available. Using this method, a single branch of the execution tree will be randomly explored.
 
-* The `dynamic_one_shot` transform is made compatible with the Catalyst compiler.
-  [(#5766)](https://github.com/PennyLaneAI/pennylane/pull/5766)
-  [(#5888)](https://github.com/PennyLaneAI/pennylane/pull/5888)
+* The `dynamic_one_shot` transform received a few improvements:
   
-* Rationalize MCM tests, removing most end-to-end tests from the native MCM test file,
-  but keeping one that validates multiple mid-circuit measurements with any allowed return
-  and interface end-to-end tests.
+  * `dynamic_one_shot` is now compatible with `qml.qjit` (the Catalyst compiler).
+    [(#5766)](https://github.com/PennyLaneAI/pennylane/pull/5766)
+    [(#5888)](https://github.com/PennyLaneAI/pennylane/pull/5888)
+  * `dynamic_one_shot` now uses a single auxiliary tape with a shot vector and `default.qubit` implements 
+    the loop over shots with `jax.vmap`.
+    [(#5617)](https://github.com/PennyLaneAI/pennylane/pull/5617)
+  * `dynamic_one_shot` is now compatible with `jax.jit`.
+    [(#5557)](https://github.com/PennyLaneAI/pennylane/pull/5557)
+
+* Mid-circuit measurement tests have been streamlined and refactored, removing most end-to-end tests 
+  from the native MCM test file, but keeping one that validates multiple mid-circuit measurements with 
+  any allowed return and interface end-to-end tests.
   [(#5787)](https://github.com/PennyLaneAI/pennylane/pull/5787)
```

Comment:
Minor, but this feels not to exciting and maybe worth moving to the bottom of this section.